src/main/java/seedu/duke/TimeTable.java

```diff
@@ -3,59 +3,70 @@
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.Calendar;
-import java.util.Locale;
```

Comment:
Should the circled part start from the base of the UI class activation bar instead of the lifeline?
I noticed that the class names are missing a colon. Should you include the colon?
![image](https://user-images.githubusercontent.com/60385689/97514782-8c211a80-19ca-11eb-95cb-a6a665bdd1b8.png)